src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7657,8 +7657,15 @@ namespace UnitarySystems {
         int CoilType_Num = this->m_SuppHeatCoilType_Num;
 
         if ((CoilType_Num == DataHVACGlobals::Coil_HeatingGasOrOtherFuel) || (CoilType_Num == DataHVACGlobals::Coil_HeatingElectric)) {
-            HeatingCoils::SimulateHeatingCoilComponents(
-                state, CompName, FirstHVACIteration, _, this->m_SuppHeatCoilIndex, _, true, this->m_FanOpMode, this->m_SuppHeatPartLoadFrac);
+            HeatingCoils::SimulateHeatingCoilComponents(state,
+                                                        CompName,
+                                                        FirstHVACIteration,
+                                                        this->m_DesignSuppHeatingCapacity * this->m_SuppHeatPartLoadFrac,
```

Comment:
In `calcUnitarySuppSystemToSP` pass the supp heat coil load argument here and several other places so the supp heating coil doesn't expect it's own setpoint node (and then generate a spurious error message about a missing setpoint).

Reply:
OK, the parent has the SP, the child can just meet a load based on PLR. Except for the iteration implication (gas and elec will just return the correct outlet temp) this is fine. What implication? Out of scope but could have called gas or elec with a SP at their outlet node (internally set) and wouldn't need to iterate. Again, this is fine.